sway/sway.5.scd

```diff
@@ -707,6 +707,22 @@ The default colors are:
 	Adjusts the blur radius of windows between 0 (disabled) and 10
 	while 5 is the default value.
 
+*blur_noise* <value>
+	Adjusts the amount of noise applied to the blur between 0 and 1 
+	while 0.02 is the default value.
+
+*blur_brightness* <value>
+	Adjusts the brightness of the blur between 0 and 2
+	while 0.9 is the default value
+
+*blur_contrast* <value>
+	Adjusts the contrast of the blur between 0 and 2
+	while 0.9 is the default value.
+
+*blur_saturation* <value>
+	Adjusts the amount of noise applied to the blur between 0 and 2
+	while 1.1 is the default value.
+
```

Comment:
nit that also applies to the README, let's clarify the units here (at least for noise, which as mentioned was a percentage)

Reply:
Amended bec8c6b